 How is Exceptional handling done in Python?
 
        # There are 3 main keywords i.e. try, except, and finally which are used to catch exceptions and handle the recovering mechanism accordingly. 
                # Try is the block of a code that is monitored for errors. 
                # Except block gets executed when an error occurs.

                # The beauty of the final block is to execute the code after trying for an error. This block gets executed irrespective of whether an error occurred or not. Finally, block is used to do the required cleanup activities of objects/variables.